### PR TITLE
feat: move to Go module

### PR DESCRIPTION
**Pull request by miton18**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
 language: go
 go:
-- 1.10.x
 - 1.11.x
 - 1.12.x
+- master
```
Comment (aurrelhebert) [inline .travis.yml]: At the moment building Noderig is failing with master...

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,14 @@
 language: go
 go:
-- 1.10.x
```
Comment (aurrelhebert) [inline .travis.yml]: Anyway to keep a build with dep on go 1.10?

File: Makefile [hunk 3]
```diff
@@ -66,8 +66,8 @@ build-go-in-docker:
 		-e GOBIN=/go/bin/ -e CGO_ENABLED=0 -e GOPATH=/go \
 		-v ${PWD}:/go/src/github.com/ovh/noderig \
 		-w /go/src/github.com/ovh/noderig \
-		golang:1.8.0 \
-			make glide-install go-build-in-docker
+		golang:1.12.0 \
```
Comment (aurrelhebert) [inline Makefile]: golang:1.12.0: only for go 1.12.0?

**Comment by FlorentinDUBOIS** [comment]
resolved with PR #61

**Pull request closed without merging**